### PR TITLE
set version via parameter to support Paralles 10

### PR DESCRIPTION
**Pull request by mklose**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -4,9 +4,9 @@
 #
 #   include parallels
 
-class parallels {
+class parallels ($version = '9') {
```
Comment (tarebyte) [inline manifests/init.pp]: do you still want the default installation of parallels to be 9?

**Comment by thisishugo** [comment]
I actually no longer use Parallels, but I expect bumping the version to use whatever the latest release version is will work for the majority of users.

File: manifests/init.pp (hunk 1 above)
Comment (mklose) [inline manifests/init.pp]: well I would prefer 10 to be default

**Comment by mklose** [comment]
I would also like to remove the way version 8 (parallels::v8) is defined. What do you think?

File: manifests/init.pp (hunk 1 above)
Comment (mklose) [inline manifests/init.pp]: the default version is set to 10 now. Is there anything else to do before you can merge?

**Comment by jacobbednarz** [comment]
@mklose if you want to fix the merge conflicts, I'm happy to merge this.

**Pull request closed without merging**